tools/CMakeLists.txt

```diff
@@ -4,7 +4,8 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 #
-#===------------------------------------------------------------------------===#
-
+#------------------------------------------------------------------------------#
```

Comment:
I'm no expert on LLVM license headers, but maybe some consistency zealot will complain about the missing equals signs.

Reply:
That's probably in several spots -- I'll look it over again (working was my main priority for this round... ;-).

Reply:
It looks like LLVM CMakeLists actually do not have licenses: see https://github.com/llvm/llvm-project/blob/master/llvm/CMakeLists.txt#L1 for instance. So maybe you can save the trouble here (but maybe someone with more LLVM knowledge than me should confirm that though).

Reply:
They don't. It's not needed.

Reply:
I'll go ahead and clean them up to match the mono-repo style...